docs/_static/vdr/v1.yaml

```diff
@@ -100,7 +100,32 @@ paths:
             text/plain:
               schema:
                 type: string
-
+    delete:
+      summary: Deactivatates a Nuts DID Document according to the specification.
+      operationId: "deactivateDID"
+      tags:
+        - DID
+      responses:
+        "200":
+          description: DID Document has been deactivated.
```

Comment:
DID or DID Document?

When there's no content returned, consider returning HTTP `201` (No Content)

Reply:
201 is created. Perhaps you mean 204? I would like to keep the set of status codes small. 200 seems a legit reply code.